flask_app/__init__.py

```diff
@@ -3,23 +3,23 @@
 from flask_app.authenticate import auth_app
 from flask_app.bookmodel import book_app
 from flask_app.category import category_app
```

Comment:
You should autoformat using isort and flake8
This link can help
https://maulik-parmar.medium.com/auto-formatting-python-code-with-black-and-isort-711e0691e631